SCons/Tool/MSCommon/common.py

```diff
@@ -365,46 +365,47 @@ def normalize_env(env, keys, force: bool=False):
     return normenv
 
 
+# TODO:  Hard-coded list of the variables that (may) need to be
+# imported from os.environ[] for the chain of development batch
+# files to execute correctly. One call to vcvars*.bat may
+# end up running a dozen or more scripts, changes not only with
+# each release but with what is installed at the time. We think
+# in modern installations most are set along the way and don't
+# need to be picked from the env, but include these for safety's sake.
+# Any VSCMD variables definitely are picked from the env and
+# control execution in interesting ways.
+# Note these really should be unified - either controlled by vs.py,
+# or synced with the the common_tools_var # settings in vs.py.
+_VS_VC_VARS = [
```

Comment:
I think you can remove the `_` prefix here. There's no major reason to sugguest accessing this from other code should be avoided ?

Reply:
It is an indicator that it shouldn't necessarily be modified.  I feel don't feel strongly either way.

We should probably add a debug logging statement for each of the keys retrieved from the list in `normalize_env` since the list can now be modified.  The mscommon log file is the easiest way to determine if a user modified the list.

Reply:
Pushed change removing the `_` prefix and added debug logging in the loop that evaluates the keys.